appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy

```diff
@@ -393,6 +393,36 @@ class UserPhotoTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
+	def 'Richard\'s photo link is still available after he updated his nickname'()
+	{
+		given:
+		User richard = addRichard()
+		def multipartEntity = MultipartEntityBuilder.create()
```

Comment:
🔧  Please use `uploadUserPhoto(richard)` below

Reply:
Much better indeed. I didn't notice that helper method.